commit f4fdb9770c76113f38515245fecc5f11b3ace20d
Author: Jan Alexander Steffens (heftig) <jan.steffens@gmail.com>
Date:   Tue Dec 4 17:54:42 2018 +0100

    fdkaacenc: Remove MODE_2_1
    
    This is not a standard mode and no longer supported by fdk-aac 2.0.0.
    
    For https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/issues/825

diff --git a/ext/fdkaac/gstfdkaacenc.c b/ext/fdkaac/gstfdkaacenc.c
index ad2bcb492..8410e684c 100644
--- a/ext/fdkaac/gstfdkaacenc.c
+++ b/ext/fdkaac/gstfdkaacenc.c
@@ -71,10 +71,6 @@ static const struct
   GST_AUDIO_CHANNEL_POSITION_FRONT_CENTER,
           GST_AUDIO_CHANNEL_POSITION_FRONT_LEFT,
           GST_AUDIO_CHANNEL_POSITION_FRONT_RIGHT}}, {
-    3, MODE_2_1, {
-  GST_AUDIO_CHANNEL_POSITION_FRONT_LEFT,
-          GST_AUDIO_CHANNEL_POSITION_FRONT_RIGHT,
-          GST_AUDIO_CHANNEL_POSITION_LFE1}}, {
     4, MODE_1_2_1, {
   GST_AUDIO_CHANNEL_POSITION_FRONT_CENTER,
           GST_AUDIO_CHANNEL_POSITION_FRONT_LEFT,
